src/languages/en.js

```diff
@@ -147,8 +147,8 @@ export default {
         openLinkInBrowser: 'open this link in your browser',
     },
     magicCodeModal: {
```

Comment:
Let's update this to `validateCodeModal` now that we've renamed `src/components/MagicCodeModal.js` to `ValidateCodeModal.js`

Reply:
good catch! Thanks!

Reply:
fixed.